activesupport/lib/active_support/duration.rb

```diff
@@ -370,6 +370,8 @@ def ago(time = ::Time.current)
     alias :before :ago
 
     def inspect #:nodoc:
+      return "0 seconds" if parts.empty?
+      
```

Comment:
Rubocop is complaining about the trailing spaces on this line, can you remove them? ✂️

Reply:
done